README.markdown

```diff
@@ -3592,6 +3593,27 @@ These constants are usually used by the [ngx.log](#ngxlog) method.
 
 [Back to TOC](#nginx-api-for-lua)
 
+Nginx escape type constants
+---------------------------
+
+**context:** *init_by_lua&#42;, init_worker_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
+
+```lua
+
+   ngx.ESCAPE_URI
+   ngx.ESCAPE_ARGS
+   ngx.ESCAPE_URI_COMPONENT
+   ngx.ESCAPE_HTML
+   ngx.ESCAPE_REFRESH
+   ngx.ESCAPE_MEMCACHED
+   ngx.ESCAPE_MAIL_AUTH
```

Comment:
Nowadays we try to avoid bloating the `ngx` table. Better avoid adding these fields. Also we don't have documentation to explain what these types really mean. Maybe we only need to document `URI` and `URI_COMPONENT` types for now.